{{cookiecutter.project_slug}}/package.json

```diff
@@ -14,8 +14,8 @@
     "url": "https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git"
   },
   "engines": {
-    "node": "10.15.0",
-    "yarn": "1.13.0"
+    "node": "{{ cookiecutter.node_version }}",
+    "yarn": "{{ cookiecutter.yarn_version }}"
```

Comment:
@wlonk This is the only piece here that concerns me. I like how you're using lts/latest as cookiecutter defaults, but I don't think they're technically correct values here in package.json. Could we pre-fill this with the current lts/latest versions on package-creation, if they selected those default options?

Reply:
Oh, maybe it's already doing that?

Reply:
Yep. It looks for `lts` or `latest` as appropriate, and pulls the version in question in the post-gen hook.